test/e2e/cloud/nodes.go

```diff
@@ -57,7 +57,7 @@ var _ = SIGDescribe("[Feature:CloudProvider][Disruptive] Nodes", func() {
 
 		newNodes, err := e2enode.CheckReady(c, len(origNodes.Items)-1, 5*time.Minute)
 		gomega.Expect(err).To(gomega.BeNil())
-		gomega.Expect(len(newNodes)).To(gomega.Equal(len(origNodes.Items) - 1))
+		framework.ExpectEqual(len(newNodes), len(origNodes.Items) - 1)
```

Comment:
hi, thanks for the PR.
is this the only instance?

Reply:
Use ag to search `gomega.Equal` and get only one match.

```go
ag "gomega.Equal"  test/e2e/cloud/**/*.go
test/e2e/cloud/nodes.go
60:		gomega.Expect(len(newNodes)).To(gomega.Equal(len(origNodes.Items) - 1))
```